eng/Versions.props

```
@@ -229,6 +229,7 @@
     -->
     <Analyzer_MicrosoftCodeAnalysisCSharpVersion>3.3.1</Analyzer_MicrosoftCodeAnalysisCSharpVersion>
     <Analyzer_MicrosoftCodeAnalysisCSharpWorkspacesVersion>3.3.1</Analyzer_MicrosoftCodeAnalysisCSharpWorkspacesVersion>
+    <RDG_MicrosoftCodeAnalysisVersion>4.5.0</RDG_MicrosoftCodeAnalysisVersion>
```

```suggestion
    <MicrosoftCodeAnalysisVersion_LatestVS>4.5.0</MicrosoftCodeAnalysisVersion_LatestVS>
```

Can we follow the same naming pattern as https://github.com/dotnet/runtime/blob/main/eng/Versions.props#LL71C6-L71C43?

---

Maybe also add a comment above this line explaining. It would be better here than in the Generator.csproj